src/pkg/path/path.go

```diff
@@ -280,6 +280,31 @@ func verifyInputValues(tenant, resourceOwner string) error {
 	return nil
 }
 
+func (pb Builder) ToServicePrefix(
+	tenant, resourceOwner string,
+	s ServiceType,
+	c CategoryType,
+) (Path, error) {
```

Comment:
why require a builder instance at all? Seems like you could just do `path.ToServicePrefix` and save some keystrokes

Reply:
Goooood question.